Tasks/CopyFilesV2/copyfiles.ts

```diff
@@ -16,8 +16,14 @@ const preserveTimestamp: boolean = tl.getBoolInput('preserveTimestamp', false);
 // normalize the source folder path. this is important for later in order to accurately
 // determine the relative path of each found file (substring using sourceFolder.length).
 sourceFolder = path.normalize(sourceFolder);
-let allPaths: string[] = tl.find(sourceFolder); // default find options (follow sym links)
-let sourceFolderPattern = sourceFolder.replace('[','[[]'); // directories can have [] in them, and they have special meanings as a pattern, so escape them
+// we allow broken symlinks - since there could be broken symlinks found in source folder, but filtered by contents pattern
+let allPaths: string[] = tl.find(sourceFolder,
+    {
+        allowBrokenSymbolicLinks: true,
+        followSpecifiedSymbolicLink: true,
+        followSymbolicLinks: true
+    });
```

Comment:
Nit: Can we extract this object and use it as the const?

Reply:
Yep sure - moved it to separate object